m2-hotfixes/README.md

```diff
@@ -0,0 +1,17 @@
+# Add custom patches
+
+You can add custom patch files to this directory. All custom patch filenames must have the `.patch` extension, for example `<patch_name>.patch`.
+After all Magento patches are applied, custom patches are applied in alphabetical order from the root of the Magento project as shown in the following example:
+
+    ```
+    diff -Naur a/vendor/package-name/ClassName.php b/vendor/package-name/ClassName.php
+    --- a/vendor/package-name/ClassName.php
+    +++ b/vendor/package-name/ClassName.php
+    ...
+    ```
+    
+You can use the following command to test the patch process–applying all Magento patches and your custom patches  locally before you build and deploy to a Cloud environment.
+``` 
+./vendor/bin/ece-tools patch
```

Comment:
This is deprecated in `2002.1`

Reply:
@shiftedreality what will be instead?

Reply:
and if we plan to deliver these changes before 2002.1 then better not to mention this not released version

Reply:
Direct call to patches `./vendor/bin/ece-patches apply`

Reply:
@YPyltiai will we deliver this earlier that minor version of ece-tools? or wait till its release and add command as for 2002.0 as for 2002.1?